### PR TITLE
added PHPDocs, removed unused variable

### PR DESCRIPTION
**Pull request by daniel-mueller**
Hi,

I added some PHPDocs and removed a unused variable.

greetings,
Daniel

### CODE REVIEW

**Comment by fzaninotto** [comment]
`@param` annotations without type and/or comments are useless IMO. Please amend your PR to add types for the phpDoc you added, or remove them.

**Comment by daniel-mueller** [comment]
Thanks for feedback. I removed every @param where I couldn't safely determine the type. On the others I added a type.

File: src/Faker/DefaultGenerator.php [hunk 1]
```diff
@@ -15,11 +15,18 @@ public function __construct($default = null)
         $this->default = $default;
     }
 
+    /**
+     * @param string $attribute
```
Comment (fzaninotto) [inline src/Faker/DefaultGenerator.php]: nah, not necessarily `string`. Same for `__call`

File: src/Faker/Factory.php [hunk 2]
```diff
@@ -25,6 +25,10 @@ public static function create($locale = self::DEFAULT_LOCALE)
         return $generator;
     }
 
+    /**
+     * @param string $locale
```
Comment (fzaninotto) [inline src/Faker/Factory.php]: why don't you document `$provider`, too? (string). Same for `findProviderClassname`

File: src/Faker/DefaultGenerator.php (hunk 1 above)
Comment (fzaninotto) [inline src/Faker/DefaultGenerator.php]: Sorry, forget that.

**Comment by daniel-mueller** [comment]
I added the two annotations.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**